<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SensorAddWindow</class>
 <widget class="QDialog" name="SensorAddWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>736</width>
    <height>261</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sensor Add Window</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="4" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>sensor_name</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="lineEdit_sensor_name"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>sensor_description</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="lineEdit_sensor_description"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>offset</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_offset"/>
   </item>
   <item row="11" column="0">
    <widget class="QPushButton" name="pushButton_CLEAN">
     <property name="text">
      <string>CLEAN</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QPushButton" name="pushButton_INPUT">
     <property name="text">
      <string>INPUT</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QLabel" name="label_input_info">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="comboBox_tool_name"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>tool_name</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="comboBox_method_name">
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>method_name</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
